relay/sources/apache/rudder-apache-relay-common.conf

```diff
@@ -75,7 +75,9 @@ Alias /reports /var/rudder/reports/incoming
 </Directory>
 
 # Load relay-api
-WSGIScriptAlias /rudder/relay-api /opt/rudder/share/relay-api/relay-api.wsgi
+#WSGIScriptAlias /rudder/relay-api /opt/rudder/share/relay-api/relay-api.wsgi
```

Comment:
In fact we need to keep this for old API not migrated to relayd yet.

Reply:
We may need to make it more precise or put the ProxyPass before to avoid conflict

Reply:
i think the proxypass below is already precise enough